splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -74,25 +76,52 @@ public class SplunkIntegration extends EndpointRouteBuilder {
     public void configure() throws Exception {
         configureErrorHandler();
         configureIngress();
+        configureIoFailed();
+        configureHttpFailed();
         configureReturn();
         configureSuccessHandler();
         configureHandler();
 
     }
 
     private void configureErrorHandler() throws Exception {
-        Processor ceEncoder = new CloudEventEncoder(COMPONENT_NAME, RETURN_TYPE);
-        Processor resultTransformer = new ResultTransformer();
         onException(IOException.class)
-            .to(direct("error"))
+            .to(direct("ioFailed"))
+            .handled(true);
+        onException(HttpOperationFailedException.class)
+            .to(direct("httpFailed"))
             .handled(true);
-        // The error handler. We set the outcome to fail and then send to kafka
-        from(direct("error"))
+    }
+
+    private void configureIoFailed() throws Exception {
+        Processor ceEncoder = new CloudEventEncoder(COMPONENT_NAME, RETURN_TYPE);
+        Processor resultTransformer = new ResultTransformer();
+        // The error handler found an IO Exception. We set the outcome to fail and then send to kafka
+        from(direct("ioFailed"))
+            .setBody(simple("${exception.message}"))
+            .setHeader("outcome-fail", simple("true"))
+            .process(resultTransformer)
+            .marshal().json()
+            .log("Failed cloud event, id ${header.ce-id}, with IO exception : ${exception.message}")
+            .process(ceEncoder)
+            .to(direct("return"));
+    }
+
+    private void configureHttpFailed() throws Exception {
+        Processor ceEncoder = new CloudEventEncoder(COMPONENT_NAME, RETURN_TYPE);
+        Processor resultTransformer = new ResultTransformer();
+        // The error handler found an HTTP Exception. We set the outcome to fail and then send to kafka
+        from(direct("httpFailed"))
             .setBody(simple("${exception.message}"))
             .setHeader("outcome-fail", simple("true"))
             .process(resultTransformer)
             .marshal().json()
-            .log("Failed cloud event, id ${header.ce-id}, with exception : ${exception.message}")
+            .log("Failed cloud event, id ${header.ce-id}, with HTTP exception : ${exception.message}")
```

Comment:
I wonder whether we can have these being shown as ERROR logs rather than INFO. @CodeHeeler do you know how?

Reply:
addressed in next push, thanks for pointing me to the correct streaming logs docs :)